x/mint/types/constants.go

```diff
@@ -24,12 +24,20 @@ const (
 	// stabilize at. In practice, TargetInflationRate acts as a minimum so that
 	// the inflation rate doesn't decrease after reaching it.
 	TargetInflationRate = 0.015
+
+	// InitialInflationRateCip29 is the inflation rate specified in CIP-29.
+	InitialInflationRateCip29 = 0.0536
+	// DisinflationRateCip29 is the rate at which the inflation rate decreases each year (after CIP-29 was introduced).
+	DisinflationRateCip29 = 0.067
 )
 
 var (
 	initialInflationRateAsDec = math.LegacyNewDecWithPrec(InitialInflationRate*1000, 3)
 	disinflationRateAsDec     = math.LegacyNewDecWithPrec(DisinflationRate*1000, 3)
 	targetInflationRateAsDec  = math.LegacyNewDecWithPrec(TargetInflationRate*1000, 3)
+
+	initialInflationRateCip29AsDec = math.LegacyNewDecWithPrec(InitialInflationRateCip29*10000, 4)
```

Comment:
Is this with precision to 4 decimal places?

Reply:
its because the number has an extra digit in it so we need one higher level of precision

Reply:
makes sense